fdbclient/include/fdbclient/BulkDumping.h

```diff
@@ -205,6 +205,7 @@ enum class BulkDumpFileType : uint8_t {
 enum class BulkDumpTransportMethod : uint8_t {
 	Invalid = 0,
 	CP = 1,
+	S3 = 2,
```

Comment:
In the following `BulkDumpState.isValid()` method, it throws `not_implemented` if transportMethod != BulkDumpTransportMethod::CP. We want to avoid throw `not_implemented` for BulkDumpTransportMethod::S3 as well.